modules/schematics/src/store/__snapshots__/index.spec.ts.snap

```diff
@@ -20,6 +20,31 @@ export class AppModule { }
 "
 `;
 
+exports[`Store Schematic should add a feature key if not root 2`] = `
```

Comment:
Extra snapshot tests because we didn't invoke the snapshot within the test.
This came to light because this gave an ESLint error.